plugins/extractors/bigquery/README.md

```diff
@@ -60,17 +60,36 @@ source:
 
 ## Outputs
 
-| Field                 | Sample Value                                                                                                                                                                                                       |
-|:----------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `resource.urn`        | `project_id.dataset_name.table_name`                                                                                                                                                                               |
-| `resource.name`       | `table_name`                                                                                                                                                                                                       |
-| `resource.service`    | `bigquery`                                                                                                                                                                                                         |
-| `description`         | `table description`                                                                                                                                                                                                |
-| `profile.total_rows`  | `2100`                                                                                                                                                                                                             |
-| `profile.usage_count` | `15`                                                                                                                                                                                                               |
-| `profile.joins`       | [][Join](#Join)                                                                                                                                                                                                    |
-| `profile.filters`     | [`"WHERE t.param_3 = 'the_param' AND t.column_1 = \"xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx\""`,`"WHERE event_timestamp >= TIMESTAMP(\"2021-10-29\", \"UTC\") AND event_timestamp < TIMESTAMP(\"2021-11-22T02:01:06Z\")"`] |
-| `schema`              | [][Column](#column)                                                                                                                                                                                                |
+| Field                          | Sample Value                                                                                                                                                                                                                                        |
```

Comment:
I don't think the contents fit well into the 2 'Field' and 'Sample Value' columns. Can we follow this structure instead - https://github.com/odpf/meteor/tree/v0.7.3/plugins/extractors/application_yaml#inputs

Reply:
Done